src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -95,6 +96,39 @@ public function getAction(Request $request, $id)
     }
 
     /**
+     * Render single container and it's widgets.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="Render single container and it's widgets.",
+     *     statusCodes={
+     *         200="Returned on success.",
+     *         404="Container not found"
+     *     }
+     * )
+     * @Route("/api/{version}/templates/containers/{id}/render", requirements={"id"="\d+"}, options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_templates_render_container")
```

Comment:
what about returning the content based on the content negotiation? If we allow to return not only json but also html, content negotiation is required in this case. I mean that this endpoint is not needed then, it can be done using the existing one but return a proper data in a proper format if requested, if the content type is not supported by the endpoint we should return a proper status code with the error. The api is inconsistent in that way because it only accepts application/json but can return also text/html. We should handle it properly imo.